### ISSUE TITLE
Documentation and tutorials

### ISSUE THREAD

**Issue by rawles** [rawles/teletext-editor#3]
It would be useful to have documentation and tutorials for this editor. One user has suggested that seeing the effects of the various control characters would be useful to those new to teletext.

**Comment by rawles** [comment]
@illarterate produced a nice [ten-minute tutorial](http://teletextart.co.uk/teletext40-editor-10-minute-tutorial/) for the editor which goes some way to implementing this suggestion.

**Comment by rawles** [comment]
@illarterate - your tutorial is not always available at the address in the README. Perhaps we could consider also having the tutorial in the repository itself.

**Comment by illarterate** [comment]
That’s true, my site has been experiencing outages recently. Simon offered to mirror it somewhere, I do believe?

Dan

**Comment by rawles** [comment]
We could just distribute the tutorial as an HTML file along with the editor.

**Comment by illarterate** [comment]
I've written a new version of the tutorial. It still isn't complete but the language is more straightforward and it incorporates the new click-to-add-character-code insertion method. I'll email it over to you now, Simon! 

Incidentally, teletextart.co.uk, which was prone to downtime, should be getting a complete overhaul (with likely T40 editor blog posts etc.) soon.

**Comment by rawles** [comment]
Thanks, I've included it in the new `doc/` folder.

**Comment by illarterate** [comment]
Excellent! I'm sure I'll work more on documentation in the coming months.

**Comment by pjfdirect** [comment]
When we did UI updates I updated Dan's PDF with new screenshots. Gissa shout if you want them

**Comment by rawles** [comment]
Any source files for the documentation would be great to have in the repository! Thanks!
